brage-import/src/main/java/no/sikt/nva/brage/migration/mapper/BrageNvaMapper.java

```diff
@@ -62,6 +63,15 @@ public final class BrageNvaMapper {
     public static final String PERSON = "person";
     public static final String BASE_PATH = new Environment().readEnv("DOMAIN_NAME");
     public static final String ORGANIZATION = "organization";
+    private static final List<String> LEGAL_NOTES_WITH_EMBARGO = List.of(
```

Comment:
Horror, jeg ser for meg at en noen har skrevet inn embargo dato som en tekstlig beskrivelse her som ikke stemmer overens med faktisk embargo dato. Så får vi kjeft etterpå.